src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/readme.metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "category": "Scene",
+  "description": "Display custom labels in a 3D scene.",
+  "formal_name": "ShowLabelsOnLayer3D",
+  "ignore": false,
+  "images": [
+    "ShowLabelsOnLayer3D.jpg"
+  ],
+  "keywords": [
+    "portal",
+    "scene",
+    "web scene"
+  ],
+  "redirect_from": [
+    "/net/latest/maui/sample-code/ShowLabelsOnLayer3D.htm"
+  ],
+  "relevant_apis": [
+    "PortalItem",
+    "Scene",
+    "SceneView"
+  ],
```

Comment:
I'd consider `LabelDefinition` and `ArcadeLabelExpression` to be very relevant here, much more so than `PortalItem`.  Adding items here will help users discover samples by adding links to the API reference.